lib/src/modules/instabug.dart

```diff
@@ -412,4 +412,9 @@ class Instabug {
     final darkKey = await dark.obtainKey(configuration);
     return _host.setCustomBrandingImage(lightKey.name, darkKey.name);
   }
+
+  /// This API has to be call when using custom app rating prompt
```

Comment:
How about we improve the documentation a little bit to add more context about the API like:
```suggestion
  /// Allows detection of app review sessions which are submitted through custom prompts.
  ///
  /// Use this when utilizing a custom app rating prompt. It should be called
  /// once the user clicks on the Call to Action (CTA) that redirects them to the app store.
  /// Helps track session data for insights on user interactions during review submission.
```